boefjes/boefjes/plugins/kat_masscan/boefje.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "masscan",
+    "name": "masscan",
+    "description": "Quickly scan large amounts of IPs.",
+    "consumes": [
+        "IPV4NetBlock"
+    ],
+    "produces": [
+        "IPAddressV4",
+        "IPAddressV6",
```

Comment:
It can't produce an `IPAddressV6` when consuming an `IPV4NetBlock`. The code to return an IPv6 address can also be removed.